src/services/index.ts

```diff
@@ -7,7 +7,24 @@ const generateAccessToken = async (config: any): Promise <any>  => {
     ...config
   });
 }
+const getOrder = async (payload: any): Promise <any>  => {
```

Comment:
This `getOrder` and `updateOrder` services are duplicate, we have defined the same in the OrderService as well. We can remove this from here.

Reply:
Removed duplicate code.